libs/event-manager/web-client/data-access/src/lib/event-api.service.ts

```diff
@@ -84,6 +96,10 @@ export class EventApiService {
   constructor(
```

Comment:
Prior to using Firebase, we had a single EventApiService, now it's a bit more complex. We need:

- EventFirebaseService: Service responsible for hitting Firebase. (currently named FirebaseService)
- EventProgramService: Service responsible for hitting the program. (currently named EventApiService)

Renaming these would be really helpful. And creating a 3rd service EventApiService for calls that need to hit both (EventFirebaseService and EventProgramService)